src/app/Fake.Core.Target/Target.fs

```diff
@@ -861,7 +861,7 @@ module Target =
             let isValidTarget name = targetLeftSet.Contains(name)
 
             let canBeExecuted (t: Target) =
-                t.Dependencies @ t.SoftDependencies |> Seq.filter isValidTarget |> Seq.isEmpty
+                t.Dependencies @ t.SoftDependencies |> Seq.exists isValidTarget |> not
```

Comment:
maybe for readability?
```suggestion
                t.Dependencies @ t.SoftDependencies |> (not << Seq.exists) isValidTarget
```

Reply:
You can't pipe it that way, doesn't compile.